packages/block-library/src/media-text/save.js

```diff
@@ -33,9 +33,29 @@ export default function save( { attributes } ) {
 		verticalAlignment,
 		imageFill,
 		focalPoint,
+		linkClass,
+		href,
+		linkTarget,
+		rel,
```

Comment:
Is it only the media that is able to be linked? If so then I'd suggest naming these new attributes so be consistent with the others:

```js
mediaLinkClass
mediaLinkHref
mediaLinkTarget
mediaLinkRel
```

Reply:
It's probably not a good idea. The attributes name are similar to what Image block is using. This will serves better when it comes to transforming between block types.

Reply:
A block transform doesn't require that attributes have the same names, e.g:

```js
transforms: {
    from: [
        {
            type: 'block',
            blocks: [ 'core/image' ],
            transform: ( { href } ) => {
                return createBlock( 'core/media-text', {
                    mediaLinkHref: href,
                } );
            },
        },
    ]
},
```

Reply:
Yeah, because they are not available yet :) Would be great to add them on transform now as well. Thanks!

Reply:
@phpbits I am not sure I understand the problem with transforms, but even so @noisysocks leaving them as they are sounds like the more consistent choice.